api/admin/controller.py

```diff
@@ -157,11 +157,29 @@ def details(self, data_source, identifier):
         if isinstance(pool, ProblemDetail):
             return pool
         work = pool.work
+
         annotator = AdminAnnotator(self.circulation)
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
 
+    def edit(self, data_source, identifier):
+        """Edit a work's metadata."""
+
+        pool = self.load_licensepool(data_source, identifier)
+        if isinstance(pool, ProblemDetail):
+            return pool
+        work = pool.work
+        changed = False
+
+        new_title = flask.request.form.get("title")
+        if new_title and work.title != new_title:
+            work.primary_edition.title = new_title
+            changed = True
+        
+        if changed:
```

Comment:
An edition has a `sort_title` as well as a `title`. Usually the `sort_title` and `title` are the same, so if the `title` changes, so should the `sort_title`.

But if `sort_title` and `title` are not the same to begin with, then changing the `title` almost always implies a change to the `sort_title`. Most of the time this change can be done automatically because it's a matter of moving "A", "An", or "The" from the beginning of `title` to the end. But this isn't always the case, and sometimes the problem is that the `title` is correct but the `sort_title` is wrong.

You need to handle all these cases:

a) sort_title and title are the same, and a change in one is reflected in the other
b) sort_title and title are the same but should not be the same
c) sort_title needs to be edited and title should stay the same, or vice versa

It's probably best to do this in a separate branch, and maybe this should even be a backlog item for now.